### PR TITLE
add janik as trusted user

### PR DESCRIPTION
**Pull request by Janik-Haag**
I don't have access to any darwin machine at the time and debugging failed builds is quite annoying to impossible without setting up hackintosh vm stuff.

A lot of people having trusted users in this repo met me in person at least once and most multiple times like raito, hexa, emilylang, 0x4a6f and lily in no particular order.

Do you need more info? Btw thank you for maintaining this :heart:

### CODE REVIEW

**Comment by RaitoBezarius** [comment]
I definitely vouch for @Janik-Haag having met them IRL :)

File: keys/janik [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAIJ4yq7oHBO2iPs4xj797a//0ypnBr27sSadKUeL2NsK6AAAABHNzaDo= janik@aq0.de
+sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAIOYg513QZsVzoyVycXZjg4F3T3+OwtcY3WAhrlfyLgLTAAAABHNzaDo= janik@aq0.de
```
Comment (mweinelt) [inline keys/janik]: No support for FIDO2 ssh keys on darwin.

File: users.nix [hunk 2]
```diff
@@ -83,6 +83,11 @@ let
       trusted = true;
       uid = 517;
     }
+    {
+      name = "janik";
+      trusted = true;
+      uid = 518;
```
Comment (0x4A6F) [inline users.nix]: You might pick `uid = 519`, because `uid = 518` will be used in #21.

File: keys/janik (hunk 1 above)
Comment (Janik-Haag) [reply keys/janik]: updated

**Review by 0x4A6F — COMMENTED** [review]
Can also vouch for Janik.

**Comment by Janik-Haag** [comment]
Sorry for the extra noise :see_no_evil: after getting access to the other builders I decided to generate my own key just for "untrusted" builders (will have to change this in the other repos at some point as well) 

oh and also *Janik now does singed commits :tada:

**Comment by Mic92** [comment]
I can also vouch for Janik.

**Pull request closed without merging**